BitcoinCore/BitcoinCore/Managers/InitialSync/InsightApi.swift

```diff
@@ -76,8 +76,8 @@ extension InsightApi: ISyncTransactionApi {
 
     class InsightTransactionOutputItem: SyncTransactionOutputItem {
         required init(map: Map) throws {
-            let script: String = try map.value("scriptPubKey.hex")
-            let address: [String] = try map.value("scriptPubKey.addresses")
+            let script: String = (try? map.value("scriptPubKey.hex")) ?? ""
+            let address: [String] = (try? map.value("scriptPubKey.addresses")) ?? []
             super.init(script: script, address: address.joined())
```

Comment:
Is it the greatest of quarter of a mileage crying like a melting water coin out of the skies from God thank you so much I thought it was never possible but I never gave up I kept going I feel like incredible hearts ready to smash let's go 2022